lib/super_digit.rb

```diff
@@ -1,15 +1,20 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - 0(n)
+# Space Complexity - 0(n)
+
 def super_digit(n)
```

Comment:
This works but it's O(d^2) where d is the number of digits.  This is because the sum is O(d) and it's inside the recursive method which goes roughly d times.